src/components/legal/TermsOfService.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Note [Non-blocking]: I don't believe it is needed to add the React import into components if they aren't using any method from the React Library. I would recommend removing it.